tests/Unit/Lti/Factory/Lti1p3RequestFactoryTest.php

```diff
@@ -87,26 +94,33 @@ public function testItReturnsAssignmentLtiRequest(): void
     {
         $expectedLtiLinkUrl = 'http://expected.url';
 
-        $ltiMessage = $this->createMock(LtiMessageInterface::class);
-        $ltiMessage
+        $this->registrationRepository
+            ->method('find')
+            ->willReturn($this->createMock(Registration::class));
+
+
+        $assignment = $this->createMock(Assignment::class);
+
+        $message = $this->createMock(LtiMessageInterface::class);
+        $message
```

Comment:
I believe you do not need this indentation

Reply:
Kept as is for now, we can discuss it with the team